osquery/main/harnesses/fuzz_utils.cpp

```diff
@@ -17,6 +17,12 @@
 
 namespace osquery {
 
+std::set<std::string> kDisabledFuzzingTables = {
+  "file",
```

Comment:
Do you want to disable plist as well? (and there are likely more that are file based)

Reply:
Yeah but these only run on Linux within oss-fuzz.